pkgs/tools/compression/lz4/default.nix

```diff
@@ -22,38 +32,17 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  # TODO(@Ericson2314): Separate binaries and libraries
-  outputs = [ "bin" "out" "dev" ];
-
-  buildInputs = lib.optional doCheck valgrind;
-
-  enableParallelBuilding = true;
+  cmakeFlags = [
+    (lib.cmakeBool "BUILD_SHARED_LIBS" enableShared)
+    (lib.cmakeBool "BUILD_STATIC_LIBS" enableStatic)
+  ];
```

Comment:
I think these aren't needed. By default it builds the shared lib, and `pkgsStatic` adds the cmakeFlag to build the static one
```suggestion
```